virttest/utils_test/__init__.py

```diff
@@ -2626,6 +2626,50 @@ def check_dest_vm_network(vm, vm_ip, remote_host, username, password,
                                   % (vm.name, results_stdout_52lts(result)))
 
 
+def check_remote_vm_disks(vm, params):
+    """
+    Check disks in remote vm are working well with I/O.
+
+    :param vm: VM object on remote host
+    :param params: the dict used for parameters
+
+    :raise: exceptions.TestFail if fail to write to the disk
+    """
+    cmd = "fdisk -l|grep '^Disk /dev'|cut -d: -f1|cut -d' ' -f2"
+    vm_ip = vm.get_address()
+    vm_pwd = params.get("password", "redhat")
+    mnt_dir = "/tmp/mnt_dir"
```

Comment:
It should be a parameter but not hard code?

Reply:
> @dzhengfy it depends on #2069 but it closed.

I'v deleted that dependence.